test/unit/alignment/pairwise/fixture/global_affine_unbanded.hpp

```diff
@@ -24,15 +24,18 @@
 namespace seqan3::test::alignment::fixture::global::affine::unbanded
 {
 
-using namespace seqan3;
-using namespace seqan3::detail;
+inline constexpr auto align_config = seqan3::align_cfg::mode{seqan3::global_alignment}
+                                   | seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-1},
+                                                            seqan3::gap_open_score{-10}}};
+inline constexpr auto align_config_dna_score = align_config
+                                             | seqan3::align_cfg::scoring{
+                                                    seqan3::nucleotide_scoring_scheme{seqan3::match_score{4},
+                                                                                      seqan3::mismatch_score{-5}}};
 
-inline constexpr auto align_config = align_cfg::mode{global_alignment} |
-                                     align_cfg::gap{gap_scheme{gap_score{-1}, gap_open_score{-10}}};
-
-static auto dna4_01 = []()
+static auto dna4_match_4_mismatch_5_gap_1_open_10_part_01 = []()
```

Comment:
Question: why is it neccessary to include the paramters in the name? It feels like this could lead to inconsistencies in the future if you change some parameters (especially since they are at the top not inside of the lambda)

EDIT: also if you are this expressive it is weird that part 01-05 are not expressive in a sense that I do not now what is testing what? I see that all tests here have the same scoring scheme anyway. It feels more helpful if the names would be `dna4_default_scoring_scheme_score_only` and then rename your `align_config_dna_score` to `align_config_dna_default_score` or something similar.

Reply:
Because I want to identify tests with the same configurations, which is necessary for testing vectorised alignments. 
So there might be added tests with different scores that cannot be put into the same vectorisation code - part01 - 05 indicates that they have the same configuration but different sequences. Not sure why this would be an issue?